federation/source-config-handlers/src/main/java/org/codice/ddf/admin/sources/wfs/WfsSourceConfigurationHandler.java

```diff
@@ -45,7 +44,7 @@
 
     @Override
     public List<ProbeMethod> getProbeMethods() {
-        return Arrays.asList(new DiscoverWfsSourceProbeMethod(), new WfsConfigFromUrlProbeMethod());
+        return Arrays.asList(new DiscoverWfsSourceProbeMethod());
```

Comment:
`Collections.singletonList()` if we want these to be immutable. I would assume we do.